SABT_MainUnit/dictionary.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * dictionary.h
+ *
+ * Created: 6/18/2015 8:45:21 AM
+ *  Author: Susan Zuo(tzuo)
+ */ 
+
+
+#ifndef DICTIONARY_H_
+#define DICTIONARY_H_
+
+typedef struct dictionary dictionary_t;
+/**
+ *  a struct to store words loaded from SD card

```

Comment:
Before you develop too much of this feature - we may want to reprioritize.  Based on the usage of SABTs at the school, I don't think this feature would be super useful to the teachers.  If they want a student to guess a specific word, they usually use two-player hangman.  For spelling many words in a sequence, the teachers have requested a Dictation Mode that would be more useful to them (email me if you need the details of that mode).  Teachers don't often use computers as part of their teaching preparations (or even use computers at all) - so it's unlikely they'll feel comfortable typing up a list of words and putting it on an SD card.  Feel free to reach out about new mode/feature projects - we want to make sure we can make really useful upgrades to the SABTs together the last couple weeks we are here and able to reprogram the SABTs!  Thanks for all the new features you've worked on so far - we're excited to test them with teachers and students soon, and will let you know how it goes.